generic/contracts/StringList.sol

```diff
@@ -0,0 +1,112 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract StringList is Ownable {
+    /// Events
+    event Add(address indexed _caller, string _value);
+    event Remove(address indexed _caller,  string _value);
+
+    string constant PLACE_HOLDER = "____INVALID_PLACE_HOLER";
+
+    /// Errors
+    string constant ERROR_VALUE_NOT_PART_OF_THE_LIST = "ERROR_VALUE_NOT_PART_OF_THE_LIST";
+    string constant ERROR_VALUE_PART_OF_THE_LIST = "ERROR_VALUE_PART_OF_THE_LIST";
+    string constant ERROR_INVALID_INDEX = "ERROR_INVALID_INDEX";
+    string constant ERROR_INVALID_VALUE = "ERROR_INVALID_VALUE";
+
+    /// State
+    string public name;
+    string[] public values;
+    mapping(string => uint256) internal indexByValue;
+
+    /**
+     * @dev Initialize contract
+     * @notice Create a new list with name `_name`
+     * @param _name The list's display name
+     */
+    constructor(string memory _name) {
```

Comment:
The owner will be initialized as the address that deploys the contract. Is that correct?

Reply:
yes, that is the expected behavior.